ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml

```diff
@@ -2771,6 +2771,72 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )images,?($|\s.*)
+  - agent: kubernetes
+    always_run: false
```

Comment:
This is the only line which was edited by hand. The rest was autogenerated with "make job"